src/pages/_error.tsx

```diff
@@ -63,8 +63,8 @@ const Error: NextPageWithLayout<Props> = ({ statusCode }: Props) => {
   )
 }
 
-Error.getInitialProps = ({ res, err }: NextPageContext) => {
-  const statusCode = res ? res.statusCode : err ? err.statusCode : 404
```

Comment:
While this _is_ slick, it's not very intuitive. I rearranged it to be a little easier on the eyes.